app/models/download.rb

```diff
@@ -212,12 +212,8 @@ def self.rank(version)
   end
 
   def self.highest_rank(versions)
-    ranks = versions.map { |version| Download.rank(version) }.reject(&:zero?)
-    if ranks.empty?
-      0
-    else
-      ranks.min
-    end
+    ranks = versions.map { |version| rank(version) }.reject(&:zero?)
+    ranks.empty? ? 0 : ranks.min
```

Comment:
@qrush :smiley: I searched a little bit in git history and first version of this method was like you said  . I don't know but maybe we just converted it to this form for specific needs. Anyway, when I was searching, I find out that this method isn't used anymore( via https://github.com/rubygems/rubygems.org/commit/9c284621a008fc69c58844cdf3beb34a8e00e0a3 ) I'm planning to delete the method and it's test, and rebased commits to one. Sounds good?